rust/kernel/error.rs

```diff
@@ -61,6 +63,26 @@ impl Error {
     }
 }
 
+impl fmt::Debug for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // SAFETY: FFI call.
+        #[cfg(CONFIG_SYMBOLIC_ERRNAME)]
+        let name = unsafe { crate::bindings::errname(-self.0) };
+        #[cfg(not(CONFIG_SYMBOLIC_ERRNAME))]
+        let name: *const c_types::c_char = core::ptr::null();
+        if name.is_null() {
```

Comment:
Should this be:
```rust
if !name.is_null() {
```
?

Reply:
You're absolutely right. I swapped the two blocks in the last minute cleanup before PR and forgot to invert the condition 🤦. Should test my change no matter how small!